app/src/main/java/net/gsantner/markor/format/markdown/MarkdownTextActions.java

```diff
@@ -71,6 +71,8 @@ public boolean runAction(String action, boolean modLongClick, String anotherArg)
             {R.string.tmaid_markdown_horizontal_line, R.drawable.ic_more_horiz_black_24dp, R.string.horizontal_line},
             {R.string.tmaid_markdown_strikeout, R.drawable.ic_format_strikethrough_black_24dp, R.string.strikeout},
             {R.string.tmaid_common_accordion, R.drawable.ic_arrow_drop_down_black_24dp, R.string.accordion},
+            {R.string.tmaid_common_deindent, R.drawable.ic_format_indent_decrease_black_24dp, R.string.deindent},
```

Comment:
As said, please don't add buttons to the bar for now, instead pursue custom order. Remove the addition and only call the `runIndentLines` in the SpecialKeys dialog.